$OpenBSD: patch-bind_bind-9_9_11_lib_isc_backtrace_c,v 1.1 2017/09/04 17:24:44 sthen Exp $

Disable backtrace support, only used on amd64 and broken with base clang
(which doesn't provide _Unwind_* yet?).

Index: bind/bind-9.9.11/lib/isc/backtrace.c
--- bind/bind-9.9.11/lib/isc/backtrace.c.orig
+++ bind/bind-9.9.11/lib/isc/backtrace.c
@@ -49,7 +49,7 @@
  */
 #ifdef HAVE_LIBCTRACE
 #define BACKTRACE_LIBC
-#elif defined(__GNUC__) && (defined(__x86_64__) || defined(__ia64__))
+#elif 0 && defined(__GNUC__) && (defined(__x86_64__) || defined(__ia64__))
 #define BACKTRACE_GCC
 #elif defined(WIN32)
 #define BACKTRACE_WIN32
